scripts/mysqld_safe.sh

```diff
@@ -17,6 +17,9 @@ MYSQLD=
 niceness=0
 mysqld_ld_preload=
 mysqld_ld_library_path=
+flush_caches=0
+numa_interleave=0
```

Comment:
Isn't mysqld_safe.sh numa_interleave option redundant due to 5.7 core server --innodb-numa-interleave?